Commands/Commands/uservoice.js

```diff
@@ -308,7 +308,7 @@ commands.info = {
               },
               timestamp: new Date(data.user.created_at)
             })
-          }).catch(() => {}) // Make bugsnag ignore it, error handling below
+          }).catch(logger.raven)
```

Comment:
I was talking about the changing the wording in the previously there comment to reflect that it was ignored on purpose in that catch. Sorry if I was a bit misleading, however this should probably be reverted to ``.catch(() => {}) // Make sentry ignore it, error handling below``